### DISCUSSION TITLE
Simplify udisk2 polkit rules

### DISCUSSION THREAD

**Discussion by fractalf** [coldfix/udiskie#259] [Show and tell]
Thank you for providing a very good starting point for me to figure out how this udisk/polkit permission stuff works!

I simplified this by using a "catch all" on `org.freedesktop.udisks2.` permissions like this:

```js
polkit.addRule(function(action, subject) {
    if (subject.isInGroup("wheel")) {
        if (action.id.startsWith("org.freedesktop.udisks2.")) {
            return polkit.Result.YES;
        }
    }
});
```

FYI :)

**Comment by coldfix** [comment]
Hey, the current recommended rules are intentionally not written as catch-all, but as a whitelist, so that people can easily enable the exact ruleset they need. Catch-all may look more convenient, but it also means that you don't need to enter password for some operations that you may not even have thought of (or that may not even have existed) at the time of installing the rule.
